release_notes/topics/11_0_0.adoc

```diff
@@ -0,0 +1,31 @@
+= Highlights
+
+== LDAPv3 password modify operation
+
+Support for LDAPv3 password modify operation was added. Also the ability in the admin console to request metadata from the configured
+LDAP server to see if it supports LDAPv3 password modify operation.
+
+Thanks to https://github.com/cachescrubber[cachescrubber]
+
+== Namespace support for LDAP group mapper
+
+Namespace support for LDAP group mapper allows to map groups from LDAP under specified branch (namespace) of the Keycloak groups tree.
```

Comment:
Should be "allows you to"

Reply:
@andymunro Thanks! This PR was already merged. So I've created new PR https://github.com/keycloak/keycloak-documentation/pull/987 and added this fix to it. Do you please have a chance to take a look at that one PR?